devfile.yaml

```diff
@@ -11,44 +11,88 @@ schemaVersion: 2.1.0
 metadata:
   name: che-code
 components:
+
   - name: dev
     container:
-      image: quay.io/che-incubator/che-code-dev:insiders
+      image: quay.io/che-incubator-pull-requests/che-code-dev:pr-253-dev-amd64
       memoryLimit: 12Gi
+      memoryRequest: 512Mi
+      cpuRequest: 500m
       cpuLimit: 3500m
+      mountSources: true
       endpoints:
         - exposure: public
           name: dev
           secure: true
           protocol: http
           targetPort: 8000
+
   - name: projects
     volume:
-      size: 3Gi
+      size: 7Gi
+
 commands:
-  - id: prepare
+  - id: yarn-prepare
     exec:
+      label: Install node dependencies
       component: dev
       workingDir: ${PROJECTS_ROOT}/che-code
       commandLine: |
         yarn prepare
       group:
         kind: build
-  - id: build
+
+  - id: yarn-build
     exec:
+      label: Compile with Yarn
       component: dev
       workingDir: ${PROJECTS_ROOT}/che-code
       commandLine: |
         yarn watch
       group:
         kind: build
         isDefault: true
-  - id: run
+
+  - id: yarn-run
     exec:
+      label: Run VS Code server on port 8000
       component: dev
       workingDir: ${PROJECTS_ROOT}/che-code
       commandLine: |
         yarn server
       group:
         kind: run
         isDefault: true
+
+  - id: podman-compile-libc
+    exec:
+      label: Compile che-code in a container using Podman
+      component: dev
+      workingDir: ${PROJECTS_ROOT}/che-code
+      commandLine: |
+        echo "Compile libc mod"
+        cp -f build/dockerfiles/linux-libc.Dockerfile build/dockerfiles/linux-libc.no-test.Dockerfile
+        REMOVE_FROM="### Testing"
+        REMOVE_TO="# Store the content of the result"
+        sed -i "/${REMOVE_FROM}/,/${REMOVE_TO}/d" build/dockerfiles/linux-libc.no-test.Dockerfile
+        podman build -f build/dockerfiles/linux-libc.no-test.Dockerfile -t linux-libc-amd64 .
+      group:
+        kind: run
+        isDefault: true
+
+  - id: podman-assemble-che-code
+    exec:
+      label: Build che-code docker image using Podman
```

Comment:
I suggest not to mention `docker` in the command label.

Reply:
@vitaliy-guliy do you see a use case when we would need to run just one of the commands `podman-compile-libc` or `podman-assemble-che-code`?

Can we introduce just a single command `Build Che-Code image` that would do everything that both these commands are doing? So, one command to get a final image with the assembly. WDYT?

Reply:
I think it's better to have them separately. If build of `che-code` image failed, you will have to compile `che-code` again.

Reply:
Label updated